packages/ariakit-utils/src/__tests__/dom-test.ts

```diff
@@ -0,0 +1,601 @@
+import {
+  closest,
+  contains,
+  getActiveElement,
+  getDocument,
+  getPopupRole,
+  getScrollingElement,
+  getTextboxSelection,
+  getWindow,
+  isButton,
+  isFrame,
+  isPartiallyHidden,
+  isTextField,
+  isVisible,
+  matches,
+  scrollIntoViewIfNeeded,
+} from "../dom";
+
+test("getDocument", () => {
+  const initialInnerHTML = document.body.innerHTML;
+  document.body.innerHTML = '<div id="testNode"></div>';
+  const node = document.getElementById("testNode");
+
+  expect(getDocument(undefined)).toEqual(document);
+  expect(getDocument(null)).toEqual(document);
+  expect(getDocument(node)).toEqual(document);
```

Comment:
I think we should use `.toBe` here:
```suggestion
  expect(getDocument(undefined)).toBe(document);
  expect(getDocument(null)).toBe(document);
  expect(getDocument(node)).toBe(document);
```